frontend/src/screens/admin/manage-learning-paths.tsx

```diff
@@ -309,6 +320,41 @@ const LearningPathForm: FC<{
                     label='Order'
                     {...form.getInputProps('order')}
                 />
+
```

Comment:
One question for @iccgoncalves is will we allow for users to select custom colors as well? or only from this predefined list.

Regardless, mantine has a nifty little color input component that you can disallow input on and just [set swatches of colors as the options](https://mantine.dev/core/color-input/#with-swatches)

We can just use that `disallowInput` prop whether or not we want them to have the ability to store custom colors. let me know what you think of this vs the combobox

Reply:
Thank you for this question @chrisbendel ;) 
We do not want to allow for custom color selection - the provided list of colors is associated with the badges color, so we want to keep it within that list. Thanks so much!

Reply:
@chrisbendel Should I continue with the combobox or use the color input component

Reply:
@Coder-Srinivas i kinda like the look of the color picker - it looks a little simpler visually. would you be willing to try it out and see how it goes?

Reply:
I think the implementation of the component will be a bit simpler too, compared to the combobox